rustsec/src/error.rs

```diff
@@ -60,11 +93,27 @@ impl Error {
 
 impl Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{}: {}", &self.kind, &self.msg)
+        let Self {
+            kind,
+            msg,
+            source: _,
+        } = self;
+        write!(f, "{kind}: {msg}")
```

Comment:
Not sure this is an improvement. Maybe if you used a couple of local bindings rather than matching on the whole struct:

```suggestion
        let kind = self.kind;
        let msg = &self.msg;
        write!(f, "{kind}: {msg}")
```

Reply:
The purpose of the `Self {}` pattern is to force a compilation error if a field is added to the struct but this code isn't updated, to avoid mistakes where a field _should_ be included in the formatting but is not. If you disprefer that, I'd rather stick to no `let`s at all. Which shall I proceed with?

Reply:
I think the original code is fine, personally